pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -342,31 +359,17 @@ func TestServiceEvents(t *testing.T) {
 		},
 	)
 	require.NoError(t, err)
+
 	msg, err := service.NewDIDCommMsg(request)
 	require.NoError(t, err)
+
 	err = didExSvc.HandleInbound(msg)
 	require.NoError(t, err)
-	// todo this function is refactored in issue-397 with update being changed. Dumming the test for this pr only
-	//	validateState(t, store, id, "responded", 100*time.Millisecond) nolint:
-}
 
-func validateState(t *testing.T, store storage.Store, id, expected string, //nolint:unused,deadcode
-	timeoutDuration time.Duration) {
-	actualState := ""
-	timeout := time.After(timeoutDuration)
-	for {
-		select {
-		case <-timeout:
-			require.Fail(t, fmt.Sprintf("id=%s expectedState=%s actualState=%s", id, expected, actualState))
-			return
-		default:
-			v, err := store.Get(id)
-			actualState = string(v)
-			if err != nil || expected != string(v) {
-				continue
-			}
-			return
-		}
+	select {
+	case <-done:
+	case <-time.After(1 * time.Second):
+		require.Fail(t, "tests are not validated")
```

Comment:
I think you mean "valid" not "validated". "not validated" basically means "not tested".

Reply:
Although "tests are not valid" also doesn't make sense. I think a more specific message here is needed. Or even just "test failed, no response..." or something like that.

Reply:
test's are valid but those were not validated due to timeout.

Reply:
Ah ok, I understand what you mean now. Consider changing the message to "validation could not be completed before timeout" or something similar.

Reply:
added 'timeout' to the message.